### PR TITLE
Adding tags and updating urls

### PR DESCRIPTION
**Pull request by seth-stage**
Adding tags for Orgs, Users, and Billing. The Stage tag still includes all endpoints.
Updating URLs in yaml to point to our alphadocs docs

### CODE REVIEW

**Review by kaiserstage — APPROVED** [review]

File: sdk-api.yaml [hunk 1]
```diff
@@ -19,18 +19,24 @@ info:
...
 
 tags:
-  - name: stage # Generated class name is based on the tag; here, it's StageApi.
+  - name: Organizations
+    description: "Our customers' organizations"
+  - name: Users
+    description: "Our customers' users"
+  - name: Billing
+    description: "The checkout session for our customers"
+  - name: Stage # Generated class name is based on the tag; here, it's StageApi.
```
Comment (dsidharta) [inline sdk-api.yaml:35]: We should probably remove this comment, or move it up above `tags`:

```
# Generated class name is based on the tag
tags:
  ...
```

**Review by dsidharta — APPROVED** [review]
Thank you 👍

File: sdk-api.yaml (hunk 1 above)
Comment (seth-stage) [reply sdk-api.yaml:35]: Sure, one sec

**Review by les-stage — APPROVED** [review]

**Review by dsidharta — APPROVED** [review]
💯

**Pull request merged**